package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "System for producing content for NDLA",
   "scripts": {
-    "build": "cross-env NODE_OPTIONS='--max-old-space-size=2048' tsc -b && razzle build --noninteractive",
+    "build": "cross-env NODE_OPTIONS='--max-old-space-size=2048' razzle build --noninteractive",
```

Comment:
Denne skal vel ikkje tas bort?

Reply:
Må fjerne typescript-sjekk ved bygg for å få now-instans til å fungere så lenge vi har liggende gamle plugins i slate som ikke er oppgradert.

Det er forøvrig en del av denne PR-en.
https://github.com/NDLANO/editorial-frontend/pull/1009